cdap-app-fabric/src/main/java/io/cdap/cdap/internal/tethering/TetheringAgentService.java

```diff
@@ -224,8 +230,12 @@ private void processTetheringControlResponse(String message, PeerInfo peerInfo)
           LOG.trace("Got keepalive from {}", peerInfo.getName());
           break;
         case RUN_PIPELINE:
```

Comment:
not from this PR, but can we rename this these to START_PROGRAM and STOP_PROGRAM?

CDAP itself only knows what programs are, it doesn't know what a pipeline is. Pipelines are a specific type of program built on top of the CDAP APIs.